editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -17,6 +39,22 @@ function MapAndLabelComponent(props: Props) {
         policyRef={props.policyRef}
         howMeasured={props.howMeasured}
       />
+      <MapContainer environment="standalone">
+        <my-map
+          drawMode
+          drawPointer="crosshair"
+          zoom={16}
+          drawFillColor={alpha(props.drawColor, 0.1)}
+          drawColor={props.drawColor}
+          drawPointColor={props.drawColor}
+          drawType={props.drawType}
+          clipGeojsonData={
+            teamSettings !== undefined
+              ? JSON.stringify(teamSettings.boundaryBBox)
+              : JSON.stringify(defaultBboxJSON)
+          }
```

Comment:
I think we could make this check a little tighter - eg right now it looks like it's possible for `teamSettings` to be definied, but `teamSettings.boundaryBBox` to not be, and for the default to never be hit? 

Maybe this would result in more intended fallback behavior here: 
```
teamSettings?.boundaryBBox 
  ? JSON.stringify(teamSettings.boundaryBBox) 
  : JSON.stringify(defaultBboxJSON)
```

Reply:
Or simply `clipGeojsonData={teamSettings?.boundaryBBox && JSON.stringify(teamSettings.boundaryBbox)}` if we forego this default per comment above? 

(Feels like ideal scenario would be handling default at `teamSettings` database level, not within this component.)

Reply:
Aw yeah, that's great! Added the default as a fallback because it was still throwing an error when I tried to return undefined, but now I see adding the ``teamSettings?`` resolves this... 

Great feedback, thanks!